<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWindow</class>
 <widget class="QWidget" name="loginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>237</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>384</width>
    <height>237</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
    font: 18px &quot;微软雅黑&quot;;
}
</string>
     </property>
     <property name="text">
      <string>聊天与文件传输工具</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="placeholderText">
            <string>账号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>密码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color:rgb(0, 174, 236);
	border:none;
	height:35px;
	border-radius: 2px;
}
QPushButton::hover{
	background-color:rgb(65, 184, 131);
}</string>
           </property>
           <property name="text">
            <string>登录</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
